cli/cli.ts

```diff
@@ -439,6 +439,8 @@ function ciAsync() {
                             .then(() => crowdin.execCrowdinAsync("upload", "built/webstrings.json"));
                     if (uploadApiStrings)
                         p = p.then(() => crowdin.execCrowdinAsync("upload", "built/strings.json"))
+                            // TODO: uncomment once verified that this is correct
```

Comment:
RE: todo here, this looks like it'd be adding to pxt-core's build, but these would typically be specific to targets, right? I believe that'd be here ish: 

https://github.com/microsoft/pxt/blob/f5a9eab92148aae6e045ae31b03aaaba708d2503/cli/crowdin.ts#L57

Reply:
This is for the skill map UI, which isn't specific to targets (just like the core pxt strings)